### ISSUE TITLE
How to set find_unused_parameters

### ISSUE THREAD

**Issue by Yoline777** [facebookresearch/detectron2#1120]
`if flag:
    loss += output(x)
`
I got error
**You can enable unused parameter detection by (1) passing the keyword argument `find_unused_parameters=True` to `torch.nn.parallel.DistributedDataParallel`;**
How to set find_unused_parameters?

**Comment by ppwwyyxx** [comment]
It is one of the "defaults" that comes with `DefaultTrainer`. If you use `DefaultTrainer` then it can't be changed.

In the future We may change it to True by default though.
